NEWS

```diff
@@ -1,3 +1,10 @@
+Flocker 1.0.1 (2015-06-XX)
```

Comment:
Not completely related but:

1. Thank you for doing the NEWS in a PR, it makes the release engineer's job easier and I think that we should add it as a contribution requirement.
2. If everyone did this and release notes and we changed to "Next Release" as a title (like with the release notes), then weekly releases could remove the review stage and be much faster.